tests/test_singleecujob.py

```diff
@@ -340,29 +340,43 @@ def test_default_lists(self):
         self.assertEqual(sej.prog_codes[0].library_refs, [])
 
     def test_resolve_references(self):
-        dl = DiagLayer(
+        diag_layer_raw = DiagLayerRaw(
             variant_type=DiagLayerType.BASE_VARIANT,
             odx_id=OdxLinkId("ID.bv", doc_frags),
             short_name="bv",
             long_name=None,
             description=None,
-            parent_refs=[],
-            communication_parameters=[],
-            services=[],
-            requests=[],
-            positive_responses=[],
-            negative_responses=[],
-            single_ecu_jobs=[self.singleecujob_object],
-            diag_comm_refs=[],
+            admin_data=None,
+            company_datas=NamedItemList(short_name_as_id),
+            functional_classes=NamedItemList(short_name_as_id, [self.context.extensiveTask]),
             diag_data_dictionary_spec=None,
-            additional_audiences=[],
-            functional_classes=[],
-            state_charts=[],
+            diag_comms=[self.singleecujob_object],
+            requests=NamedItemList(short_name_as_id, []),
+            positive_responses=NamedItemList(short_name_as_id),
```

Comment:
Is it `NamedItemList(short_name_as_id, [])` or `NamedItemList(short_name_as_id)` ?

Reply:
these are synonyms. I'll change this to `NamedItemList(short_name_as_id)`. stay tuned.

Reply:
okay, done: ee14065b56a8815b51e76745340bf75af8a1a6b0 . (I know, this should go into a separate PR, but this would mean twice the work: Once for fixing `main`, once for fixing this PR...)